python/test/t_PhysicalSpaceCrossEntropyImportanceSampling_std.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+from openturns.testing import assert_almost_equal
+ot.RandomGenerator.SetSeed(1)
+
+distribution_R = ot.LogNormalMuSigma(300.0, 30.0, 0.0).getDistribution()
+distribution_F = ot.Normal(75e3, 5e3)
```

Comment:
I guess you mean a floating point number, not an integer here.
```suggestion
distribution_F = ot.Normal(75.0e3, 5.0e3)
```